_data/simple-icons.json

```
@@ -13839,6 +13839,11 @@
             "hex": "002244",
             "source": "https://www.united.com"
         },
+        {
+            "title": "United Nations",
+            "hex": "009FDC",
+            "source": "https://commons.wikimedia.org/wiki/File:Logo_of_the_United_Nations_(B%26W).svg"
```

```suggestion
            "source": "https://www.un.org/en/",
            "guidelines:" "https://www.un.org/styleguide/pdf/UN_brand_identity_quick_guide_2020.pdf"
```
The logo is available as SVG at the en part of the website itself and the guidelines are also there.